spec/support/factory_girl.rb

```diff
@@ -5,6 +5,12 @@
     sequence(:email) {|n| "user#{n}@example.com" }
     password 'password'
     password_confirmation 'password'
+
+    factory :user_with_photo do
+      avatar { Rack::Test::UploadedFile.new(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.